src/zerocoin_v3.cpp

```diff
@@ -355,6 +355,10 @@ bool CheckSigmaMintTransaction(
     if (hasCoin) {
        LogPrintf("CheckSigmaMintTransaction: double mint, tx=%s\n",
                 txout.GetHash().ToString());
+        return state.DoS(100,
```

Comment:
Are we sure we want function CheckSigmaMintTransaction to fail, in case it's called twice?

Reply:
Yes I see this function can be called twice, it can be at miner. let me add a condition before return.